### ISSUE TITLE
Configs are not loaded in production environment

### ISSUE THREAD

**Issue by amkisko** [palkan/anyway_config#58]
## What did you do?
Run Rails and Sidekiq in production.

## What did you expect to happen?
Eager loaded config classes.

## What actually happened?
Classes are not loaded, Sidekiq jobs that use configs throw exceptions, Rails console in Sidekiq environment does not have classes loaded.

## Additional context
This workaround helped:
```
config.eager_load_paths << Rails.root.join("config", "configs")
```

## Environment
production

**Ruby Version:**
2.6.0

**Framework Version (Rails, whatever):**
Rails 5.2.4.2

**Anyway Config Version:**
2.0.1

**Comment by palkan** [comment]
Thanks for the report.
It seems I can reproduce this in tests (#59), so it's gonna be fixed soon)

**Comment by palkan** [comment]
Released a fix in 2.0.2. Check please.

**Comment by amkisko** [comment]
Thanks for super quick fix! :tada:
